aws-lambda-crawler/samconfig.toml

```diff
@@ -2,8 +2,9 @@ version = 0.1
 
 [sam-config-env.deploy.parameters]
 stack_name = "aws-lambda-crawler"
-resolve_s3 = true
-s3_prefix = "aws-lambda-crawler"
+	# We specify --s3-bucket explicitly in CI/deploy scripts; disable resolve_s3
```

Comment:
@dzunhowz Not sure why we have some `space` change in this file, could you please check this?